impl/src/main/java/io/kroxylicious/testing/kafka/common/Utils.java

```diff
@@ -227,7 +227,9 @@ private static boolean isRetryable(Throwable potentiallyWrapped) {
         // If the throwable originates from within a CompletionStage's handler, it will be wrapped within a
         // CompletionException.
         var throwable = potentiallyWrapped instanceof CompletionException && potentiallyWrapped.getCause() != null ? potentiallyWrapped.getCause() : potentiallyWrapped;
-        return throwable instanceof RetriableException || throwable instanceof InvalidReplicationFactorException
+        boolean retriable = throwable instanceof RetriableException
+                && (throwable.getMessage() == null || !throwable.getMessage().contains("The AdminClient is not accepting new calls"));
```

Comment:
I don't see a nicer way to actually discover that the AdminClient is closed.

Reply:
I had wondered if we should just throw away the admin client and create a new one...

Reply:
No, It is the caller's responsibility to pass the Admin, so I don't think it would be appropriate for createTopic to spontaneously create a new one. 

The caller is awaitExpectedBrokerCountInClusterViaTopic.  I think we can infer that awaitExpectedBrokerCountInClusterViaTopic has timed-out, and the try-with-resources has closed the Admin client.    In the case where Admin client becomes close, we want createTopic to give up.